assets/scss/custom/_overrides.scss

```diff
@@ -109,3 +109,164 @@ lite-youtube {
+      padding-bottom: 0;
+    }
+  }
+
+  &.default-pr {
+    .press-release-cards {
+      .views-row {
+        @include media-breakpoint-up(lg) {
+          width: 25%;
+        }
+
+        &:not(:first-child) {
+          @include media-breakpoint-down(lg) {
+            display: none;
+          }
+          padding-bottom: 40px;
+        }
+
+        &:first-child {
+          @include media-breakpoint-up(lg) {
+            width: calc(50% - 6rem);
+            border-right: 1px solid black;
+            padding-right: 3rem;
+          }
+
+          a {
+            font-size: 2rem;
+            line-height: 2.406rem;
+
+            @include media-breakpoint-down(lg) {
+              font-size: 1.625rem;
+              line-height: 1.95rem;
+            }
+          }
+          .body p {
+            @include media-breakpoint-up(lg) {
+              font-size: 1.5rem;
+              line-height: 2.25rem;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  &.featured-pr {
+    .press-release-cards {
+      .views-row:last-child {
+        display: none;
+      }
+      > div:has(.field_featured_post) {
+        @include media-breakpoint-up(lg) {
+          width: calc(50% - 6rem);
+          border-right: 1px solid black;
+          padding-right: 3rem;
+        }
+      }
+      > div:has(.field_latest_news_view) {
+        @include media-breakpoint-down(lg) {
+          display: none;
+        }
+        flex: 1;
+        padding-left: 3rem;
+      }
+      .field_featured_post {
+        a {
+          font-size: 2rem;
+          line-height: 2.406rem;
+
+          text-decoration: none;
+
+          &::after {
+            content: '→';
+            margin-left: 0.25rem;
+          }
+
+          &:hover {
+            text-decoration: underline;
+          }
+
+          @include media-breakpoint-down(lg) {
+            font-size: 1.625rem;
+            line-height: 1.95rem;
+          }
+        }
+        .body p {
+          @include media-breakpoint-up(lg) {
+            font-size: 1.5rem;
+            line-height: 2.25rem;
+          }
+        }
+      }
+    }
+  }
+
+  .container {
+    position: relative;
+
+    @include media-breakpoint-down(md) {
+      max-width: 100%;
+    }
+  }
+
+  .btn-outline-secondary {
+    @include media-breakpoint-up(lg) {
+      position: absolute;
+      bottom: 0;
+      right: 0;
+    }
+
+    @include media-breakpoint-down(lg) {
+      width: 100%;
+    }
+
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    justify-content: center;
+
+    &::after {
+      content: "→";
+    }
+  }
+
+  .viewsreference--view-title {
+    display: none;
+  }
+
+  .field--name-field-latest-news-view {
+    .attachment_1 {
+      display: none;
+    }
+  }
+
+  img {
+    max-width: 100%;
+    height: auto;
+    width: 100%;
+
+    @include media-breakpoint-up(md) {
+      border-radius: 1rem;
+    }
+  }
+
+  .field_caption {
+    display: none;
+  }
+}
```

Comment:
@kmurphychi247 This could break a lot of other things image related things.  We have a caption field, why are we hiding it?